### PR TITLE
Fix multiple split INCOME cash flow

### LINKED ISSUE #861: Wrong income calculation for multiple splits
In the detail page for bank accounts the total income calculation is wrong for a transaction with multiple incomes. ~I suspect the same may happen with the expenses.~ This seems to happen to any type of transaction with multiple splits

Try:

Bank +100
Income1 -75
Income2 -25

The total Income reflected is 200 instead of 100

This seems duplicate of #857 because also the sankey diagram is showing wrong quantities

### PR DESCRIPTION
**Pull request by argaen**
Fixes #861

### CODE REVIEW

File: src/__tests__/hooks/api/useCashFlow.test.tsx [hunk 1]
```diff
@@ -389,7 +389,86 @@ describe('useCashFlow', () => {
...
 
   /**
-   * The above case is simple however, when
+   * Same as the test before but with receiving money into the ASSET
+   * account instead of withdrawing
+   *
+   * As an example, say we have a tx with the following:
+   *  - 100 in account1
+   *  - -75 in account2
+   *  - -25 in account3
```
Comment (argaen) [inline src/__tests__/hooks/api/useCashFlow.test.tsx:398]: Before the change this was returning

-100 account2
-100 account3

File: src/hooks/api/useCashFlow.ts [hunk 2]
```diff
@@ -46,12 +46,14 @@ export function useCashFlow(
             ) AS mnemonic,
             SUM(
               CASE
-                WHEN splits.guid = credit_split.guid THEN
+                WHEN splits.guid = asset_split.guid THEN
```
Comment (argaen) [inline src/hooks/api/useCashFlow.ts:49]: credit_split was incorrect because for income transactions this means we are selecting the wrong account. Changed the logic to select asset type account (except INVESTMENT) which keeps all the tests passing and also passes the newly added one (plus correct behavior). Whenever we discover a new edge case can always update
